src/test/java/com/uid2/admin/job/jobsync/keyset/TestReplaceSharingtypesWithSitesJob.java

```diff
@@ -41,7 +41,7 @@ public class TestReplaceSharingtypesWithSitesJob {
     GlobalScope siteScope = new GlobalScope(globalSiteMetadataPath);
 
     List<Site> sites = List.of(
-            new Site(3, "site 3", true,  new HashSet<>()),
+            new Site(3, "site 3", true),
```

Comment:
can remove the "new HashSet<>" arguments below

Reply:
No this was the thing I was explaining about how java is being weird about overloading the constructors.
Me and Ian both added new Set but his is a Set<String> and mine is a Set<ClientType> but for whatever reason Java doesn't look at generic type of the Set in the parameter, so we can't have two constructors with only 4 parameters.